dataline-config/src/main/resources/json/SingerMessage.json

```diff
@@ -21,7 +21,8 @@
       "type": "string"
     },
     "record": {
-      "description": "record message: no schema, just a map of string to object!"
+      "description": "record message: must me a JSON object",
```

Comment:
haha. annoying that we can't apply our *Json naming convention here. oh well.

Reply:
Will actually do it in a future PR. Found a way to do so